### PR TITLE
Cache gems in travis, update links and opscode -> chef

### PR DESCRIPTION
**Pull request by tas50**
Misc fixes.  Cache gem installs in Travis to speed up testing, update links to be https, update Opscode -> Chef Software.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ You can find the answers to additional frequently asked questions [on the wiki](
 
 ## Quick-contribute
 
-*   Create an account on our [bug tracker](http://tickets.opscode.com/browse/KNIFE)
+*   Create an account on our [bug tracker](https://tickets.opscode.com/browse/KNIFE)
```
Comment (btm) [inline CONTRIBUTING.md]: this should be the Github Issues link

**Comment by btm** [comment]
:+1: but you should update the JIRA links to Github issues too.

File: CONTRIBUTING.md (hunk 1 above)
Comment (tas50) [inline CONTRIBUTING.md]: I'm going to replace that file next with 90% chef/chef's contributing doc

**Comment by gbish** [comment]
Hello,

Here is an article that I found by chance, but it turned out so interesting that I decided to share it with you, you can find it here <http://hophyshungo.parkerdentistryfacialrejuvenationandwellness.com/oiolikwu>

Kind regards, hebgb@blueyonder.co.uk

**Pull request merged**